### DISCUSSION TITLE
Images without labels are considered in the training?

### DISCUSSION THREAD

**Discussion by LorenzoCausa** [facebookresearch/detectron2#4401] [Q&A] [answered]
Are images without labels considered in the training? If not, is there a way to add unlabeled images to reduce false positives?
For example, if I want to recognize rails, I don't want my model to recognize ladders as rails. To do this I would like to give a ladder image in the training data and would not label it.

**Comment by ExtReMLapin** [comment, marked as answer]
Yes it's ignored by default, please use `cfg.DATALOADER.FILTER_EMPTY_ANNOTATIONS = True`
